src/pages/official-links.md

```diff
@@ -0,0 +1,83 @@
+| Candy Guard        | Guard1JwRhJkVH6XZhzoYxeBVQe872VH6QggF4BWmS9g | [Link](https://github.com/metaplex-foundation/mpl-candy-machine/tree/main/programs/candy-guard)             |
+| Candy Machine v3   | CndyV3LdqHUfDLmE5naZjVN8rBZz4tqhdefbAnjHG3JR | [Link](https://github.com/metaplex-foundation/mpl-candy-machine/tree/main/programs/candy-machine-core)      |
+| Core               | CoREENxT6tW1HoK8ypY1SxRMZTcVPm7R94rH4PZNhX7d | [Link](https://github.com/metaplex-foundation/mpl-core)                                                     |
+| Core Candy Guard   | CMAGAKJ67e9hRZgfC5SFTbZH8MgEmtqazKXjmkaJjWTJ | [Link](https://github.com/metaplex-foundation/mpl-core-candy-machine/tree/main/programs/candy-guard)        |
+| Core Candy Machine | CMACYFENjoBMHzapRXyo1JZkVS6EtaDDzkjMrmQLvr4J | [Link](https://github.com/metaplex-foundation/mpl-core-candy-machine/tree/main/programs/candy-machine-core) |
+| Inscriptions       | 1NSCRfGeyo7wPUazGbaPBUsTM49e1k2aXewHGARfzSo  | [Link](https://github.com/metaplex-foundation/mpl-inscription)                                              |
+| Hydra              | hyDQ4Nz1eYyegS6JfenyKwKzYxRsCWCriYSAjtzP4Vg  | [Link](https://github.com/metaplex-foundation/mpl-hydra)                                                    |
+| Token Auth Rules   | auth9SigNpDKz4sJJ1DfCTuZrZNSAgh9sFD3rboVmgg  | [Link](https://github.com/metaplex-foundation/mpl-token-auth-rules)                                         |
+| Token Metadata     | metaqbxxUerdq28cj1RbAWkYQm3ybzjb6a8bt518x1s  | [Link](https://github.com/metaplex-foundation/mpl-token-metadata)                                           |
+
+## Tokens and Nfts
+
+| Name                                             | ID                                           |
+| ------------------------------------------------ | -------------------------------------------- |
+| MPLX Token                                       | METAewgxyPbgwsseH8T16a39CQ5VyVxZi9zXiDPY18m  |
+| Metaplex Whitepaper NFT (collection address)     | 9q2Ejt1Qubk5t4f3xXaiwRtvjhHrEK8r9EYsH6gwnt39 |
+| Candy Studio NFT (collection address)            | 3NauNKfqAbQVhAJyv464Tq14jynV6THSNiGxhf7W6fP9 |
+| MPLX Token Launch Party NFT (collection address) | BodU7rcs75cf3J94EBK7vKuxoepgz6ayb2t7oJDbXtWX |
+
+## Metaplex Rulesets
+
+To be used with the pNfts Auth Rules.
+
+| Name                         | ID                                           |
+| ---------------------------- | -------------------------------------------- |
+| Metaplex Foundation Rule Set | eBJLFYPxJmMGKuFwpDWkzxZeUrad92kZRC5BJLpzyT9  |
+| Compatability Rule Set       | AdH2Utn6Fus15ZhtenW4hZBQnvtLgM1YCW2MfVp7pYS5 |
+
+<!-- ## Team Members
+
+TonyBoyle.sol - tonyboyle - 612983988646313984
+MarkSackerberg - marksackerberg - 331688811186618370
+StephenHess - StephenHess#3571 - 735124917498151022
+Blockiosaurus - gumbercules - 339220158331748352
+Urbentom - urbentom - 354305194982506499 -->
```

Comment:
Shouldn't we just remove this?

Reply:
Was potentially going to leave them in, it was something I was going to bring up. I guess we can take them out for now since I have them stored in Discord.